lib/plugin/instance.rb

```diff
@@ -292,9 +292,9 @@ def register_upload_in_use(&block)
 
   # Registers a category custom field to be loaded when rendering a category list
   # Example usage:
-  #   register_category_list_preloaded_category_custom_fields("custom_field")
-  def register_category_list_preloaded_category_custom_fields(field)
-    CategoryList.preloaded_category_custom_fields << field
+  #   register_preloaded_category_custom_fields("custom_field")
+  def register_preloaded_category_custom_fields(field)
```

Comment:
Did we check if any plugin is using the old method?

Reply:
This is a new method and all places surround it with `defined?`. This should cause no issue at all.